apps/skilavottord/ws/src/app/modules/accessControl/accessControl.service.ts

```diff
@@ -1,69 +1,132 @@
+import { AccessControlModel } from './accessControl.model'
+import { LOGGER_PROVIDER } from '@island.is/logging'
 import { Inject, Injectable } from '@nestjs/common'
-
 import type { Logger } from '@island.is/logging'
-import { LOGGER_PROVIDER } from '@island.is/logging'
-
-import { Role } from '../auth'
-import { AccessControlModel } from './accessControl.model'
+import { InjectModel } from '@nestjs/sequelize'
 import {
   CreateAccessControlInput,
   DeleteAccessControlInput,
   UpdateAccessControlInput,
 } from './accessControl.input'
+import { RecyclingPartnerModel } from '../recyclingPartner/recyclingPartner.model'
 
 @Injectable()
 export class AccessControlService {
-  constructor(@Inject(LOGGER_PROVIDER) private logger: Logger) {}
+  constructor(
+    @InjectModel(AccessControlModel)
+    private accessControlModel: typeof AccessControlModel,
+    @Inject(LOGGER_PROVIDER) private logger: Logger,
```

Comment:
Keep the same code style for injections as above... there's no inline decorators in this application.
```typescript
    @InjectModel(AccessControlModel)
    private accessControlModel: typeof AccessControlModel,
    @Inject(LOGGER_PROVIDER)
    private logger: Logger,
    ```